src/main/java/seedu/address/logic/commands/MakeStatsCommand.java

```diff
@@ -1,22 +1,57 @@
 package seedu.address.logic.commands;
 
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.chart.PieChart;
+import seedu.address.commons.core.index.Index;
 import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+import seedu.address.model.person.Person;
 
 /**
- * Format full help instructions for every command for display.
- */
+* Generates statistics for the event specified
+*/
 public class MakeStatsCommand extends Command {
 
     public static final String COMMAND_WORD = "makeStats";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Generates the Statistics of the Event at the user"
+        + "specified index. Type of statistics shown is based off a user input parameter.\n"
+        + "Parameters INDEX " + PREFIX_TYPE + "TYPE. \n"
```

Comment:
We should omit the fullstop here to maintain consistency with the other commands! 
```suggestion
        + "Parameters INDEX " + PREFIX_TYPE + "TYPE \n"
```

Reply:
thx for catching!